src/Php73/Php73.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Symfony\Polyfill\Php73;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php73
+{
+    const NANO_IN_SEC = 1000000000.0;
+
+    /**
+     * @param mixed $var
+     *
+     * @return bool
+     */
+    public static function is_countable($var)
+    {
+        return is_array($var)
+            || $var instanceof \Countable
+            || $var instanceof \ResourceBundle
+            || $var instanceof \SimpleXmlElement;
+    }
+
+    /**
+     * @param bool $asNum
+     *
+     * @return array|float
+     */
+    public static function hrtime($asNum = false)
+    {
+        if ($asNum) {
+            return microtime(true) * self::NANO_IN_SEC;
```

Comment:
This overflows 100% of the time.
What's the unit of the native function?
Does this really return a number in nanoseconds (ignoring the floating part)?
(same below btw)
This should be double checked because right now I don't get how this is supposed to work.

Reply:
@nicolas-grekas As of [PHP source](https://github.com/php/php-src/blob/1cdf5b7b4b27f35140d88d629d2977c9833384e2/ext/standard/hrtime.c#L163) it
> Returns an array of integers in form [seconds, nanoseconds] counted from an arbitrary point in time [...]
This example https://3v4l.org/Fne2W shows, that the value changes when called at different time.
Can you post an example output of this function overflowing?

Reply:
@nicolas-grekas bump :)

Reply:
I need a few more days before being able to look again at it :)

Reply:
Sure, take your time :+1: